docs/Quality.md

```diff
@@ -6,16 +6,47 @@ title: "Tallwave Guidelines — Quality"
 [&larr; Home]({{ site.root }}/)
 
 # Quality
+There is a delicate balance between doing great work and actually shipping something. We can take years to build something with a beautiful and clever architecture and no technical debt, but it would be a misuse of our clients' budgets and time to do so. Likewise, we can sub-contract out to the cheapest off-shore bidders and ship something missing key features that crashes when you look at it funny.
 
-## Tests
+We do believe it is possible to ship Good code in a reasonable amount of time, without gouging on price.
+
+## Testing
+This is our first line of defense. While it is possible to go overboard with testing zealotry, there is great value in _automated_ tests that run against your code. Tests fall into different categories. 
 
 ### Unit Tests
+A unit test examines a single function or method (a single "unit"). For example, the tests for a function called `addIntegers` would make sure that the results of that function are the sum of the arguments passed in. These are the easiest to write, but also provide the least amount of utility. More bugs occur when the units meet, or to use the proper nomenclature, when they integrate with each other.
 
 ### Integration Tests
+A proper unit test stubs out or mocks dependencies on the function. An integration test runs through 2 or more of those units and checks to see if the result is what was expected.
```

Comment:
'.. if the result is what is expected.' Tests continue to run and the result must continue to be what is expected at the time of running the test. (semantics: past or present tense).